### ISSUE TITLE
Q447 root_keywords SELECT and FROM are not right aligned

### ISSUE THREAD

**Issue by ghost** [pgjones/flake8-sql#9]
Hi!
I have this query:
```sql
CREATE OR REPLACE VIEW registration_list AS
         SELECT col1, col2, col3, col4, col5, col6, col7,
                col8, col9, SUBSTRING(col1 FROM 1 FOR 8) AS col10
              FROM table
           WHERE col5 IS NULL WITH CHECK OPTION
```
and the flake-8 kept giving me the error:
**Q447 root_keywords SELECT and FROM are not right aligned**
I tried many possibilities but i'm not able to remove the error, i would appreciate if you can help me in pointing the issue.

**Comment by ghost** [comment]
@pgjones can you help me out in this?

**Comment by pgjones** [comment]
I think it is confused by the `FROM` in the `SUBSTRING` (although in the example the `FROM` isn't right aligned with the `SELECT`). I'm unsure how to fix this right now, so I'd add a `# noqa Q447` to the line it notes as having the error.

**Comment by ghost** [comment]
so that's an issue right now? And yes this is what i thought as well because upon removing the SUBSTRING it was working fine. So we will let this issue remain open?

**Comment by pgjones** [comment]
This should be fixed by d1a1000e870db0c3fd88cf5e5c7a62c33aad51d8 and released in 0.4.0
